File: server/Models/Keep.cs
namespace keepr.Models;

public class Keep : DBItem
{
  // public int Id { get; set; }
  // public string CreatorId { get; set; }
  public string Name { get; set; }
  public string Description { get; set; }
  public string Img { get; set; }
  public bool IsPrivate { get; set; }
}

File: client/src/models/Keep.js
import { DataModel } from "./DataModel.js"

export class Keep extends DataModel {
  constructor(data) {
    // this.id = data.id
    // this.creatorId = data.creatorId
    // this.img = data.img
    // this.name = data.name
    // this.description = data.description
    this.views = data.views
    this.kept = data.kept
    this.vaultKeepId = data.vaultKeepId
    // this.creator = data.creator
  }
}